app/validator.js

```diff
@@ -80,7 +88,7 @@ var validator = {
         });
 
         result = Joi.validate(reqBody, reportSchema);
-        return result;
+        return preprocessor(result);
```

Comment:
is there any reason to have a method that is used only in one place? 
if yes, then maybe the method name could be renamed to have something more meaningful?